drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
Thanks for the advice! I will begin implementing the following API right away:

```
   /**
    * Adds the robot models specified in a file to this RigidBodySystem.
    * Note that this method only adds models whose dynamics are enabled.
    * Static models like ground planes or height maps will need to be
    * added separately by calling getRigidBodyTree()->addCollisionElement().
    *
    * @param file_name The name of the file containing the robot model(s).
    * @param floating_base_type The type of the joint used in the
    * foating virtual 6 DoF that connects the robot models to the world.
    * By default, this is DrakeJoint::QUATERNION.
    * @param pose_of_model_in_world Transform giving the pose of the file's
    * model frame expressed in the world frame.
    * @return The number of robot models added.
    */
   int AddRobotsFromFile(
      const std::string& file_name,
      const DrakeJoint::FloatingBaseType floating_base_type =
          DrakeJoint::QUATERNION,
      const Eigen::Isometry3d & pose_of_model_in_world
          = Isometry3d::Identity());
```
@sherm1: I'm really looking forward to your visit. In addition to getting all of the terminology formalized, I'm hoping to get some advice on how to setup a Drake development environment in Windows.